src/features/announcements/Announcement.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext } from "react";
+import { Button } from "components/ui/Button";
+import { Context } from "features/game/GameProvider";
+import { announcements } from "features/announcements";
+import fence from "assets/land/goblin_fence.png";
+import { useShowScrollbar } from "lib/utils/hooks/useShowScrollbar";
+import classNames from "classnames";
+
+const CONTENT_HEIGHT = 400;
+export const Announcement: React.FC = () => {
+  const { ref: itemContainerRef, showScrollbar } =
+    useShowScrollbar(CONTENT_HEIGHT);
+  const { gameService } = useContext(Context);
+
+  const storedDate = localStorage.getItem("announcementLastRead");
+
+  function onAcknowledge() {
+    gameService.send("ACKNOWLEDGE");
+  }
+
+  const filtered = storedDate
+    ? announcements.filter(
+        (announcement) => announcement.date > new Date(storedDate)
+      )
+    : announcements;
+
+  // Sort latest first
+  filtered.sort((a, b) => b.date.getTime() - a.date.getTime());
+
+  return (
+    filtered && (
+      <div
+        ref={itemContainerRef}
+        style={{ maxHeight: CONTENT_HEIGHT }}
+        className={classNames("overflow-y-auto", {
+          scrollable: showScrollbar,
+        })}
+      >
+        <p className="flex flex-col items-center text-2xl mb-4">
```

Comment:
A few suggestions around the UI of this component.

1. The "What's new" is too big and draws immediate attention. I would suggest making the feature name this large size - (e.g. "Chickens")
2. Most people don't like to read (and some players don't understand English). Consider putting the feature image above the text (see screenshot below for an example). This will help capture their attention.
3. Assume the users will quickly scan these so they can continue to farm. We only have a short period to grab their attention and convey the information we are after. In this case here I would suggest using bullet point format as it is easier to digest and scan for relevant information

<img width="538" alt="Screen Shot 2022-06-12 at 9 17 49 am" src="https://user-images.githubusercontent.com/11745561/173207928-bafcfcbd-9a40-4409-afe1-e680ad8747e1.png">